File: solutions/beecrowd/2437/2437.cs
using System;

class Solution {
    static void Main() {
        int x_m, y_m, x_r, y_r;
        string line;
        string[] numbers;

        while ((line = Console.ReadLine()) != null) {
            numbers = line.Split(' ');

            x_m = Convert.ToInt32(numbers[0]);
            y_m = Convert.ToInt32(numbers[1]);
            x_r = Convert.ToInt32(numbers[2]);
            y_r = Convert.ToInt32(numbers[3]);

            Console.WriteLine("{0}", Math.Abs(x_r - x_m) + Math.Abs(y_r - y_m));
        }
    }
}

File: solutions/beecrowd/2437/2437.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var x_m, y_m, x_r, y_r float64

	for {
		_, err := fmt.Scanf("%f %f %f %f", &x_m, &y_m, &x_r, &y_r)
		if err != nil {
			break
		}

		fmt.Printf("%.0f\n", math.Abs(x_r-x_m)+math.Abs(y_r-y_m))
	}
}
